src/NuGet.Services.EndToEnd/Support/Clients/GalleryClient.cs

```diff
@@ -156,6 +157,23 @@ public async Task UnlistAsync(string id, string version, ITestOutputHelper logge
             await SendToPackageAsync(HttpMethod.Delete, id, version, logger);
         }
 
+        public async Task SearchPackageODataV2FromDBAsync(string id, string normalizedVersion, ITestOutputHelper logger)
+        {
+            var galleryEndpoint = await GetGalleryUrlAsync(logger);
+            var url = $"{galleryEndpoint}/api/v2/Packages()?$filter=tolower(Id) eq '{id.ToLower()}' and NormalizedVersion eq '{normalizedVersion}' and 1 eq 1";
+            using (var httpClient = new HttpClient())
```

Comment:
Can you use `_httpClient ` instead?

Reply:
The test is very simple I do not need the _httpClient.

Reply:
I’m not sure I follow. What’re the benefits of not using `_httpClient`?

Reply:
Talked offline